token/hmac/hmacsha.go

```diff
@@ -19,7 +19,7 @@
  *
  */
 
-// Package hmac is the default implementation for generating and validating challenges. It uses HMAC-SHA256 to
+// Package hmac is the default implementation for generating and validating challenges. It uses HMAC-SHA512_256 to
```

Comment:
The correct naming is `SHA-512/256`

```suggestion
// Package hmac is the default implementation for generating and validating challenges. It uses SHA-512/256 to
```

There are also a couple more places in this document that can be updated :)